src/App.tsx

```diff
@@ -15,36 +15,90 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Alphabet = 'Alphabet',
+  Length = 'Length',
+  Default = '',
+}
+
+function prepareGoods(goods: string[], type: SortType, reverse: boolean) {
```

Comment:
1. use arrow func here better
2. rename  prepareGoods ->  normalizeGoods and reverse -> reversed